### ISSUE TITLE
The webpage link of downloading the three datasets is invalid.

### ISSUE THREAD

**Issue by KonkonYoumu** [yoyo-yun/MA-Bert#3]
Hello,I find that the link of downloading datasets is invalid now.Could you please update the link?Thank you very much!

**Comment by drunkprogrammer** [comment]
Yes, I keep the dataset. The link is https://pan.baidu.com/s/1iewWK8ZKgxsQQg497lGvBQ?pwd=ca9r
Password：ca9r

**Comment by KonkonYoumu** [comment]
> Yes, I keep the dataset. The link is https://pan.baidu.com/s/1iewWK8ZKgxsQQg497lGvBQ?pwd=ca9r Password：ca9r

Thank you very much! :D
